Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
+    system "go", "build", "-o", "bin/slacknimate", "github.com/mroth/slacknimate"
+    bin.install "bin/slacknimate"
+  end
+
+  test do
+    system "#{bin}/slacknimate", "--version"
+    system "#{bin}/slacknimate", "--help"
```

Comment:
For this particular program, I unfortunately couldn't figure one out that was significant.  Since all of this program's functionality involves speaking to an external API (for which is needed a valid API token to be passed), the only other thing that could be tested here is it's throwing of an error condition on absence of the user providing the token, but checking for an exit code of 1 versus a success condition didn't seem like an established pattern in brew formula (I can add a test for this if it is desired however, or if anyone can think of something better!).